src/openvic-simulation/military/UnitInstanceGroup.cpp

```diff
@@ -173,7 +173,7 @@ bool UnitInstanceManager::generate_unit_instance(
 		[&unit_deployment]() -> UnitInstanceBranched<Branch> {
 			if constexpr (Branch == UnitType::branch_t::LAND) {
 				// TODO - get pop from Province unit_deployment.get_home()
-				return { unit_deployment.get_name(), unit_deployment.get_type(), nullptr };
+				return { unit_deployment.get_name(), unit_deployment.get_type(), nullptr, false };
```

Comment:
Could you make this `false` a constant named "is_not_mobilised" or something similar?

Reply:
I'll add a comment next to the argument, and move the pop comment next to `nullptr` similarly.